browser.js

```diff
@@ -395,6 +395,71 @@ function write (buffer, string, offset, length, encoding) {
   return codecFor(encoding).write(buffer, string, offset, length)
 }
 
+function writeDoubleLE (buffer, value, offset) {
+  offset ??= 0
+
+  const view = new DataView(buffer.buffer)
```

Comment:
```suggestion
  const view = new DataView(buffer.buffer, buffer.byteOffset, buffer.byteLength)
```

A typed array is a view over an underlying buffer, possibly only containing a subset of it. The data view must reflect this so that `offset` remains relative to `buffer.byteOffset`.